README.md

```diff
@@ -1 +1,7 @@
-# best-repo-ever
\ No newline at end of file
+# best-repo-ever
```

Comment:
thought of giving it 107. But removed it as I couldn't do things the right way. I will learn to

Reply:
May be it would be great if I get some clear explanation